debian/rules

```diff
@@ -11,7 +11,9 @@ DISABLE_JEMALLOC_ARCH_LIST := hppa m68k
 DEB_HOST_ARCH ?= $(shell dpkg-architecture -qDEB_HOST_ARCH)
 
 # Set local state dir for FHS
-LOCAL_CONFIGURE_FLAGS = --localstatedir=/var/lib --libdir=/usr/lib
+LOCAL_CONFIGURE_FLAGS  = --localstatedir=/var/lib --libdir=/usr/lib
+LOCAL_CONFIGURE_FLAGS += --with-sphinx-build=sphinx-build-3
```

Comment:
On Debian, the command is "sphinx-build", no matter which package is installed. I can't find a "sphinx-build-3" command.

If both python-sphinx and python3-sphinx is installed (no matter which order), we get:
`/usr/bin/sphinx-build -> ../share/sphinx/scripts/python2/sphinx-build`

I have no idea why it defaults to 2, or how to configure it...

Reply:
The reason why I did this is because there was no /usr/bin/sphinx-build when I ran builds for stretch.

Not sure what went wrong.